src/app/auth/forgot-password/forgot-password.component.ts

```diff
@@ -25,14 +25,12 @@ export class ForgotPasswordComponent implements OnInit {
     this.auth.initiateResetPassword(this.email).subscribe(val => {
       this.done = true;
     }, error => {
-      console.log(error);
-      this.error(error.error);
+      this.error(error);
```

Comment:
Is error a string or an object?

Reply:
In this case it's a string. Unlike in CARD the error message is stored in error.